net/sscep/Makefile

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (C) 2006 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sscep
+PKG_VERSION:=0.10.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v${PKG_VERSION}.tar.gz
+PKG_SOURCE_URL:=https://github.com/certnanny/sscep/archive/
+PKG_HASH:=489cc8e093986776eb3f15082bf766778f707176f3cd604bf0ef1008da06b8e5
+
+PKG_LICENSE:=OpenSSL
+PKG_MAINTAINER:=John Marrett <johnf@zioncluster.ca>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/sscep
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+libopenssl +openssl-util
+  TITLE:=A Simple SCEP (Simple Certificate Enrollment Protocol) client
+  URL:=https://github.com/certnanny/sscep/
+endef
+
+define Build/Prepare
+$(call Build/Prepare/Default)
+	cd $(PKG_BUILD_DIR); ./bootstrap.sh
+endef
+
+define Build/Configure
+	cd $(PKG_BUILD_DIR); ./configure
+endef
+
+define Build/Compile
+	$(MAKE) -C $(PKG_BUILD_DIR) \
```

Comment:
It's better to switch to use CMake.

Reply:
How can I do this? Can you please point me to an example package?